app/src/main/java/ar/com/wolox/android/cookbook/instagramlogin/InstagramLoginRecipeView.kt

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.cookbook.instagramlogin
+
+import ar.com.wolox.android.cookbook.instagramlogin.model.InstagramDataItem
+
+interface InstagramLoginRecipeView {
+
```

Comment:
I'm used to put a whiteline in between methods. I don't know what @lmiotti thinks?

Reply:
Yes, I agree with that

Reply:
ok, i will change in "login-twitter" branch after made the changes in this PR